superset-frontend/plugins/plugin-chart-ag-grid-table/package.json

```diff
@@ -27,6 +27,8 @@
     "@react-icons/all-files": "^4.1.0",
     "@types/d3-array": "^2.9.0",
     "@types/react-table": "^7.7.20",
+    "ag-grid-community": "^33.1.1",
```

Comment:
should this be in `peerDependencies`? Also wondering if we need to rethink where the main deps should live and how the peerDeps should be set. I think what makes more sense is to add the main dep in `@superset-ui/core` and have the main package/app AND plugins set them as peerDep.